js_modules/dagster-ui/packages/ui-core/src/runs/TimeElapsed.tsx

```diff
@@ -12,8 +12,8 @@ export const TimeElapsed = (props: Props) => {
   const {startUnix, endUnix} = props;
 
   const [endTime, setEndTime] = React.useState(() => (endUnix ? endUnix * 1000 : null));
-  const interval = React.useRef<NodeJS.Timer | null>(null);
-  const timeout = React.useRef<NodeJS.Timer | null>(null);
+  const interval = React.useRef<NodeJS.Timeout | null>(null);
```

Comment:
I realize this wasn't added in this PR but why is this NodeJS.Timeout ? We're not running this code in NodeJS

Reply:
Yeah, agree, this is a little weird. I think it *should* be `number`, but maybe there's something we need to do in our tsconfig to make that behave properly. I'll see if I can fix it in a followup.